javascript/ql/src/semmle/javascript/TypeScript.qll

```diff
@@ -1474,6 +1474,41 @@ class ExternalModuleScope extends @externalmodulescope, Scope {
   override string toString() { result = "external module scope" }
 }
 
+/**
+ * TypeScript import declared in an explicit augmentation to global, 
+ * in the form of a variable of typeof the import.
+ * For example:
+ * import * as net_outer from "net"
+ * declare global {
+ * 		var net: typeof net_outer
+ * }
+ * 
+ * var s = net.createServer(); // this reference to net is an import
+ * 
+ * This import pattern was observed when analyzing the package web-component-tester
+ * https://www.npmjs.com/package/web-component-tester
```

Comment:
This comment shouldn't be part of the qldoc, it can either go in the PR description or a corresponding JIRA issue (if there is one).